src/raftstore/store/worker/pd.rs

```diff
@@ -176,6 +184,50 @@ impl<T: PdClient> Runner<T> {
             error!("report split failed {:?}", e);
         }
     }
+
+    fn handle_get_region(&self, local_region: metapb::Region, peer: metapb::Peer) {
+        metric_incr!("pd.get_region");
+        match self.pd_client.get_region(local_region.get_start_key()) {
```

Comment:
This is bound to PD interface/impl. And it will change a lot to add the feature of region merge, e.g., the raftstore internals, PD region info indexing ... Without a whole picture of how region merge is designed and implemented, maybe it's not wise to change PD to be prepared for this incoming feature.

We could hold and rethink/merge this PR(and the changes to PD) until the whole region merge design comes out sometime in the future.